### PR TITLE
updates to readme with recent versions

### PR DESCRIPTION
**Pull request by motobob**

### CODE REVIEW

**Comment by motobob** [comment]
after this going to tag it, ship it , ship it to HEX and PR to erlcloud

**Comment by nalundgaard** [comment]
While you're editing it, why not add a build status link?

File: README.rst [hunk 1]
```diff
@@ -28,35 +28,25 @@ Result form::
 History
 =======
 
-current
+1.2.4
 -------
+- Unicode everywhere
```
Comment (nalundgaard) [inline README.rst]: technically just in values. Also, we added support for dashes in keys and section titles.

Comment (motobob) [reply README.rst]: addressed

**Comment by nalundgaard** [comment]
👍 LGTM

**Pull request merged**